doc/source/user_guide/ug_running.rst

```diff
@@ -837,6 +834,47 @@ modify the scripts and input settings in the case directory, NOT the run directo
 In general, files in the run directory are overwritten by versions in the case
 directory when the model is built, submitted, and run.
 
+.. _aliases:
+
+Useful aliases
+-------------------
+
+This section provides a list of some potentially useful aliases that leverage the CICE 
+scripts.  These are not defined by CICE and are not required for using CICE.  They
+are provided as a sample of the types of things that can be done by users.
+
+The current **ice_in**, **cice.settings**, and **env.[machine]** files are copied from 
+the case directory into the run directory when the model is run.  Users can create aliases 
+leveraging the variables in these files.  Aliases like the following can be established 
+to quickly switch between case and run directories and query or carry out other tasks
+quickly:
+
+.. code-block:: bash
+
+  #!/bin/tcsh
+  # From a case or run directory, source the necessary environment files to run CICE
+  alias cice_env 'source env.*; source cice.settings'
+  # Go from case directory to run directory and back (see https://stackoverflow.com/a/34874698/)
+  alias cdrun  'set rundir=`\grep "setenv ICE_RUNDIR" cice.settings | awk "{print "\$"NF}"` && cd $rundir'
+  alias cdcase 'set casedir=`\grep "setenv ICE_CASEDIR" cice.settings | awk "{print "\$"NF}"` && cd $casedir'
+
+  #!/bin/bash
+  # From case/test directory, go to run directory
+  alias cdrun='cd $(cice_var ICE_RUNDIR)'
+  # From run directory, go to case/test directory
+  alias cdcase='cd $(cice_var ICE_CASEDIR)'
+  # monitor current cice run (from ICE_RUNDIR directory)
+  alias cice_tail='tail -f $(ls -1t cice.runlog.* |head -1)'
+  # open log from last CICE run (from ICE_CASEDIR directory)
+  alias cice_lastrun='$EDITOR $(ls -1t logs/cice.runlog.* |head -1)'
+  # open log from last CICE build (from ICE_CASEDIR directory)
+  alias cice_lastbuild='$EDITOR $(ls -1t logs/cice.bldlog.* |head -1)'
+  # show CICE run directory when run in the case directory
+  alias cice_rundir='cice_var ICE_RUNDIR'
+  # open a tcsh shell and source env.* and cice.settings (useful for launching CICE in a debugger)
+  alias cice_shell='tcsh -c "cice_env; tcsh"'
+
```

Comment:
The  `cdcase`, `cdrun` and `cice_rundir` aliases as written above need the following shell function:

```suggestion
# Print the value of a CICE variable ($1) from cice.settings
cice_var() {
\grep "setenv $1" cice.settings | awk "{print "\$"3}"
}
```

Reply:
I didn't notice that.  I just cut and paste some of the aliases.  Personally, I think these aliases are a bit complicated for many users.  But I will try to fix this.

Reply:
Well, users do not have to understand how the aliases work to benefit from them :)

Reply:
I added the cice_var alias to the documentation.